ui/pages/desktop-pairing/desktop-pairing.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import testData from '../../../.storybook/test-data';
+import configureStore from '../../store/store';
+import DesktopPairingPage from './desktop-pairing.component';
+
+const store = configureStore(testData);
```

Comment:
Ah I see. If you can pass the generateDesktopOtp as a prop then you don't need the Provider or redux stuff. All stories are passed this by default. It's only if you need to adjust mock redux state.

```suggestion
import DesktopPairingPage from './desktop-pairing.component';

```

Reply:
Make sense, removed.